src/main/java/seedu/address/model/transaction/TransactionLog.java

```diff
@@ -56,4 +59,16 @@ public boolean isEmpty() {
         return transactionList.isEmpty();
     }
 
+    /**
+     * Creates an observable list of transactions to be used by MainWindow
+     * @return an unmodifiableObservableList of transactions
+     */
+    public ObservableList<Transaction> listTransactions() {
```

Comment:
I suggest for a change in this method's name. `listTransactions()` is a verb and denotes that an action will happen, but it is just returning the transactions as an ObservableList. As such, in the MainWindow, 
```java
ObservableList<Transaction> transactions = company.getTransactions().listTransactions();
```
it looks odd. I think we should follow the method name in `UniquePocList` and call it `asUnmodifiableObservableList` or something better.

Reply:
Thank you! I have changed the method name